### ISSUE TITLE
Select a timezone and show local time next to profile username

### ISSUE THREAD

**Issue by MaybeThisIsRu** [hometown-fork/hometown#1231]
### Pitch

An optional timezone setting that can be inferred from the browser using the new Internationalization API if the user opts into it. Then a small chip is shown next to username to indicate their current local time.

Something like:

<img width="371" alt="image" src="https://user-images.githubusercontent.com/3974019/205428705-8b2e4276-0572-4eb3-8d52-571f8acf2200.png">

### Motivation

SmallCamp has members from all over the world. Sometimes it's helpful knowing what time it is somewhere just to get a sense of their current reality through the day, send a timezone appropriate greeting, or not make an ask that's odd for their time.

**Comment by MaybeThisIsRu** [comment]
This seems unnecessary... I will admit, but it's also kinda fun and could be a cool + somewhat useful feature for a social network to implement. And ofc it should be opt in. :)

**Comment by dariusk** [comment]
I think the combination of two factors makes me have to turn down this feature request.

1. it is, as you say, a bit of a nice-to-have rather than necessary (but I do see the value!)
2. I have done a lot of work with timezone stuff in my day job and frankly it is a neverending nightmare, even using the (great) new browser internationalization APIs. the last thing I want for hometown work for me is to have to field timezone bugs.

basically it's a mental-health-for-me-as-maintainer thing! Closing this but feel free to keep commenting here, I'm open to re-opening if a lot of people want this feature
